db/schema.rb

```diff
@@ -305,12 +305,20 @@
   add_index "photos", ["status_message_guid"], name: "index_photos_on_status_message_guid", length: {"status_message_guid"=>191}, using: :btree
 
   create_table "pods", force: :cascade do |t|
-    t.string   "host",       limit: 255
+    t.string   "host",          limit: 255
     t.boolean  "ssl"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.